packages/paste-core/components/combobox/src/styles/ComboboxListbox.tsx

```diff
@@ -23,12 +23,12 @@ const ComboboxListbox = React.forwardRef<HTMLUListElement, ComboboxListboxProps>
       boxShadow="shadow"
       listStyleType="none"
       margin="space0"
-      maxHeight="size50"
       overflowY="auto"
       padding="space0"
       paddingY="space30"
       position="absolute"
       width="100%"
+      maxHeight="260px"
```

Comment:
Curious why `260px`?

Reply:
So for infinite scroll to work correctly, we need to make the popout have the height of `numItems * itemHeight`, so that the scrollbar reflects the total set and not the 10 or so `li`s that are actually rendered. Doing so makes the height huge on long lists, so the actual popout is set to a max-height and we provide an inner `li` that is presentational with the total height to fake it.

That said, the maxHeight here was hardcoded as something that looked goodish, but I swapped it to a token now (size30) which is similar in PX value.